### ISSUE TITLE
Size marker images to be larger than the marker itself

### ISSUE THREAD

**Issue by floogulinc** [mn-pollinators/buzz-about#17]
This doesn't appear to be as easy as we thought.

When we add a marker:
```ts
//Create the THREEAR BarcodeMarker object for this marker
let barcodeMarker = new THREEAR.BarcodeMarker({
  barcodeValue: marker.barcodeValue,
  markerObject: markerGroup,
  size: 1,
  minConfidence: 0.2
});
```

Changing `size` to a larger value, like 5, makes it smaller. Changing it to a fraction, like 0.5, causes errors and doesn't show anything.

**Comment by michael-small** [comment]
When I put it to 0.5, it looks the same size as 1 but it sticks to the center of the marker layer until it becomes undetected.

**Comment by michael-small** [comment]
**Is this basically closed?**

Paul's solution posted on Slack:

> `let geometry1 = new THREE.PlaneBufferGeometry(2,2, 1,1)`

**Comment by helloworld12321** [comment]
Oh, you know, we actually closed this one with #81!
